ui/src/views/access-point/nfa-picker.tsx

```diff
@@ -8,6 +8,7 @@ import { AppLog } from '@/utils';
 import { CreateAccessPoint } from './create-ap.context';
 
 export const NfaPicker: React.FC = () => {
+  //TODO: remove

```

Comment:
Remove? 😆

Reply:
lol sorry forgot to remove this component, good catch